spring-boot-dependencies/pom.xml

```diff
@@ -135,6 +135,7 @@
 		<spring-amqp.version>1.6.0.M1</spring-amqp.version>
 		<spring-cloud-connectors.version>1.2.1.RELEASE</spring-cloud-connectors.version>
 		<spring-batch.version>3.0.6.RELEASE</spring-batch.version>
+		<spring-data-geode.version>1.7.0.APACHE-GEODE-EA-M1</spring-data-geode.version>
```

Comment:
As I explained [here](https://spring.io/blog/2015/06/12/spring-data-gemfire-supports-apache-geode) and [here](https://groups.google.com/a/pivotal.io/d/msg/spring-developer/KMbUfmlS_jA/F_EiftPFBwAJ), this is not a usual version number.

Essentially, 2 separate releases of _Spring Data GemFire_ are occurring off 1 codebase (in GitHub) right now.  Though, the _Spring Data GemFire Support for Apache Geode_ is technically occurring off a branch (`apache-geode`).

Based on the issues I had with the starter, I think now, even more than ever/before, it is time to consider _Spring Data GemFire with Apache Geode Support_ as a separate project (SD module) having a separately releasable artifact (i.e. `spring-data-geode`) and intuitive version number (i.e. `1.0.0.M1`).